tools/ExamplesGenerator.ps1

```diff
@@ -98,24 +121,84 @@ function Get-ExternalDocs-Url {
         [ValidateSet("beta", "v1.0")]
         [string] $GraphProfile = "v1.0",
         [string] $UriPath,
+        [string] $Module = "Users",
+        [string] $GenerationMode = "auto",
+        [string] $ManualExternalDocsUrl,
         [ValidateNotNullOrEmpty()]
         [string] $Command = "Get-MgUser",
         [Hashtable] $OpenApiContent,
+        [System.Object] $Method = "GET",
         [string] $GraphProfilePath = (Join-Path $PSScriptRoot "..\src\Users\Users\examples\v1.0")
     )
+    $MissingExternalDocsUrlFolder = Join-Path $PSScriptRoot "..\openApiDocs\MissingExternalDocsUrl\$Module.csv"
```

Comment:
@timayabi2020, who is this file meant for? Should it be in the docs or metadata repo instead? External docs link is used by all SDKs and GE, not just PowerShell. Limiting this file to just PowerShell may not be ideal.

Reply:
@peombwa the report can be shared by just directing someone to this repository upon request.  Since we are the ones maintaining the reports, which happens after every weekly example generation pipeline run., I don't see how we are limiting the report to just PowerShell. Also, the pipeline runs after the weekly open Api docs refresh pipeline and this ensures that the reports are up to date.

Reply:
I see. We should then put the report in https://github.com/microsoftgraph/msgraph-metadata/tree/master/openapi. The sliced OpenAPI documents in this repo are only used by PowerShell. Missing external docs is beyond the scope of this repo as it originates from the metadata and cuts across all SDKs.

To centralize where we track issues, I'd say let's move the report generation to the metadata repo, where it is actionable. Ideally, we should run the report for the entire MS Graph API surface area, which only exists in https://github.com/microsoftgraph/msgraph-metadata/tree/master/openapi. This can be an independent AzDo pipeline in the metadata repo that runs after metadata to OpenAPI conversion.

Reply:
As discussed, I have removed the responsibility of generating the missing external docs report from that script.